recipes/bonsu/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "Bonsu" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d10e491325e5ff63e6a9d2aa8574cd25961e5796aa9ee666c50f24d27a80a5b6
+  patches:
+    # submitted upstream as https://github.com/bonsudev/bonsu/pull/1
+    - setup.py.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv --global-option=build_ext --global-option=-I$PREFIX/include"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ cdt('libXxf86vm') }}  # [linux]
+    - {{ cdt('mesa-libgl') }}  # [linux]
+  host:
+    - pip
+    - setuptools
+    - python
+    - pillow
+    - wxpython
+    - vtk
+    - numpy
+    - {{ mpi }}  # [mpi != 'nompi']
+    - fftw
+    - fftw * mpi_{{ mpi }}_*  # [mpi != 'nompi']
+  run:
+    - python
+    - pillow
+    - wxpython
+    - vtk
+    - numpy
+    - {{ mpi }}  # [mpi != 'nompi']
+    - fftw
```

Comment:
So it appears the docs don't have this `fftw` in the runtime env. Maybe remove it?